beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -78,6 +84,23 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
             return;
         }
 
+        if self.parent_lookups.iter_mut().any(|parent_req| {
+            parent_req.contains_block(&hash) || parent_req.add_peer(&hash, &peer_id)
```

Comment:
```suggestion
             parent_req.add_peer(&hash, &peer_id) || parent_req.contains_block(&hash)
```
I didn't think hard enough if these functions are mutually exclusive. If they are, then this suggestion is pointless. I think they are, but I've started writing this now. 

If they are not, we prob want to add this peer in, so should be prioritized.

Reply:
so right now I only add peers that claim the current requested parent, not peers claiming the previous ones. So, as is, these two calls are disjoint. But I think it's safe to add peers that reference one of the downloaded blocks then? What do you think

Reply:
oh yeah I see. Cool to leave it as is :)